### ISSUE TITLE
Cannot run kubefwd 

### ISSUE THREAD

**Issue by famani-omny** [txn2/kubefwd#1]
When running kubefwd I get the following error 

```
panic: No Auth Provider found for name "gcp"

goroutine 1 [running]:
github.com/txn2/kubefwd/cmd/kubefwd/services.glob..func1(0x17db920, 0xc4202ee440, 0x0, 0x2)
	/Users/cjimti/go/src/github.com/txn2/kubefwd/cmd/kubefwd/services/services.go:74 +0x1545
github.com/txn2/kubefwd/vendor/github.com/spf13/cobra.(*Command).execute(0x17db920, 0xc4202ee420, 0x2, 0x2, 0x17db920, 0xc4202ee420)
	/Users/cjimti/go/src/github.com/txn2/kubefwd/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/txn2/kubefwd/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc4202f2500, 0xc4202f2500, 0xc4202f2780, 0x17db920)
	/Users/cjimti/go/src/github.com/txn2/kubefwd/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/txn2/kubefwd/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4202f2500, 0x1, 0x1)
	/Users/cjimti/go/src/github.com/txn2/kubefwd/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/Users/cjimti/go/src/github.com/txn2/kubefwd/cmd/kubefwd/kubefwd.go:64 +0x67
```

**Comment by cjimti** [comment]
Seems to have something to do with the client lib I am using not supporting gcp. I don't have anything running on GCP at the moment so I'll have to set up a test soon and try it out.

I assume kubectl works normally for you? Any other info for me to reproduce would be appreciated. Thanks.

**Comment by famani-omny** [comment]
Yes kubectl works normally for me. I can port-forward pods individually no problem, but I can't use your utility to port-forward all the pods in a namespace.

**Comment by imduffy15** [comment]
https://github.com/txn2/kubefwd/pull/2 should fix this.

**Comment by cjimti** [comment]
@imduffy15 thanks for the pull request! @famani-omny #2 is merged and a new version 1.3.0 should be available. I don't have any easy way to test it at the moment so any feedback would be appreciated.

**Comment by famani-omny** [comment]
Worked correctly with no issues.
